features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/sha/sha256_alt_sw.h

```diff
@@ -42,8 +35,7 @@ extern "C" {
 /**
  * \brief          SHA-256 context structure
  */
-typedef struct
-{
+typedef struct {
     uint32_t total[2];          /*!< number of bytes processed  */
     uint32_t state[8];          /*!< intermediate digest state  */
     unsigned char buffer[64];   /*!< data block being processed */
```

Comment:
Consider using a `#define` for `64` here (and re-use it in the C file).

Reply:
@Patater Same reason as above.